### ISSUE TITLE
Train a support vector machine classifier

### ISSUE THREAD

**Issue by covuworie** [covuworie/nobel-physics-prizes#79]
Train a linear vector machine classifier using the original features and the topics features with and without the sample weights. Select the following hyperparameters based on maximum MCC on the validation set:

- Penalty parameter `C` of the error term. Trades off correct classification of training examples against maximization of the decision function’s margin. For larger values of `C`, a smaller margin will be accepted if the decision function is better at classifying all training points correctly. A lower `C` will encourage a larger margin, therefore a simpler decision function, at the cost of training accuracy. In other words `C` behaves as a regularization parameter in the SVM.
- `class_weight`, the weights associated with the classes.

Fix the `kernel` parameter as *linear* since this is only kernel for which weights of separating hyperplane are available.
